src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -143,10 +144,23 @@ const Expander = ({ expanded }: ExpandProps) => {
   return <ExpandIcon $expanded={expanded} />
 }
 
-export function Trade({ trade, outputUSDC, gasUseEstimateUSD, expanded }: TradeProps & TradeTooltip & ExpandProps) {
+export function Trade({
+  trade,
+  outputUSDC,
+  gasUseEstimateUSD,
+  expanded,
+  loading,
+}: TradeProps & TradeTooltip & ExpandProps) {
   return (
     <>
-      <Caption caption={<Price trade={trade} outputUSDC={outputUSDC} />} />
+      <Caption
+        caption={
+          <ThemedText.Body2 color="secondary" opacity={loading ? 0.4 : 1}>
```

Comment:
I don't think you need to specify color here, i think it will inherit, but i may be wrong